conode/Makefile

```diff
@@ -21,10 +21,15 @@ flags=-ldflags="$(ldflags)"
 
 all: docker
 
+# Use this target to build from only published sources.
 docker: Dockerfile
 	BUILDFLAG=$(flags)
 	docker build -t $(IMAGE_NAME):$(VERSION) --build-arg BUILDFLAG="$(ldflags)" .
 
+# Use this target to build from local source instead of from publish sources.
+docker_dev: Dockerfile-dev exe/conode.Linux.x86_64
```

Comment:
I'm not sure this is what you want:

https://stackoverflow.com/questions/9159960/order-of-processing-components-in-makefile

So according to this, it would first create the Dockerfile-dev, then the binary. Which sounds counter-intuitive...